.codeclimate.yml

```diff
@@ -1,42 +1,26 @@
 ---
-engines:
+version: "2"
+plugins:
   shellcheck:
     enabled: true
-  checkstyle:
-    enabled: true
-    channel: "beta"
   csslint:
     enabled: true
   duplication:
     enabled: true
     config:
-      enabled: true
-      threshold: 7
       languages:
-        - typescript
```

Comment:
Thanks to the updated yml file, it was relatively quick for me to test how many duplications we would get. Here are my findings: 
- current: 20
- 45 (default): 18
- 57: 13

Just ignore this comment. When I started writing, I thought there was no change. And at this point, I didn't want to delete it.